Dockerfile.bitbucket_pipeline

```diff
@@ -0,0 +1,11 @@
+FROM python:3.10 as base
+
+WORKDIR /app
+ADD pyproject.toml .
+ADD requirements.txt .
+RUN pip install . && rm pyproject.toml requirements.txt
+ENV PYTHONPATH=/app
+ADD pr_agent pr_agent
+ADD bitbucket_pipeline/entrypoint.sh /
+RUN chmod +x /entrypoint.sh
+ENTRYPOINT ["/entrypoint.sh"]
```

Comment:
**Suggestion:** Consider using a multi-stage build to reduce the size of the final Docker image.
```suggestion
# First stage: build
FROM python:3.10 as builder

WORKDIR /app
ADD pyproject.toml .
ADD requirements.txt .
RUN pip install . && rm pyproject.toml requirements.txt

# Second stage: runtime
FROM python:3.10-slim as runtime

COPY --from=builder /app /app
ENV PYTHONPATH=/app
ADD pr_agent pr_agent
ADD bitbucket_pipeline/entrypoint.sh /
RUN chmod +x /entrypoint.sh
ENTRYPOINT ["/entrypoint.sh"]
```

Reply:
that won't work, no dependencies in "runtime" target. But I suggest moving this to the docker/Dockerfile as a new target, follow the other examples in this file @sarbjitsinghgrewal